deployment/kubernetes/helm/ros-ocp/values.yaml

```diff
@@ -7,6 +7,14 @@ global:
   pullPolicy: IfNotPresent
   # Pull secrets for private registries
   imagePullSecrets: []
+  # Init container images (using alternative registries to avoid docker.io rate limits)
+  initContainers:
+    waitFor:
+      repository: registry.access.redhat.com/ubi9/ubi-minimal
```

Comment:
i am not certain it is a good idea to specify the registry. what if I want to deploy without access to redhat?